ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -98,6 +99,7 @@ int enigma_main(int argc, char** argv) {
 
 namespace enigma_user {
 
+int os_browser = browser_not_a_browser;
```

Comment:
Why isn't this const? Who's going to update it?

Reply:
I didn't make it constant because we haven't made a lot of the other global constants actually constant, so I was unsure. Anyway, the GMS manual does state that it's constant, so: c4a9dd0e10392c79023f3943ccbfc271cf9a484e

There, I've made it constant here too.